### PR TITLE
Auth work

### PR DESCRIPTION
**Pull request by Aram-Anderson**
#

Description of Changes Made: 
Added auth based views to navbar
Anticipated Code Effects: 
(Would this merge affect any other piece of our code?)
None
Reviewer: 
@djtrujillo @snayrouz @sidewinder2020

### CODE REVIEW

File: app/controllers/store/admin/store_controller.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class Store::Admin::StoreController < ApplicationController
   def edit
+
     @store = Store.find_by(slug: params[:store_slug])
   end
 
   def update
-    @store = Store.find_by(slug: params[:store_slug]
+    @store = Store.find_by(slug: params[:store_slug])
```
Comment (snayrouz) [inline app/controllers/store/admin/store_controller.rb:8]: good find.

File: app/services/permission_service.rb [hunk 2]
```diff
@@ -41,7 +40,7 @@ def authorized?
       return true if controller == "store/admin/users" && action.in?(%w(index update show))
       return true if controller == "dashboard" && action.in?(%w(index))
       return true if controller == "store/items" && action.in?(%w(index destroy create new update edit))
-      return true if controller == "store/admin/store" && action.in?(%w(update))
+      return true if controller == "store/admin/store" && action.in?(%w(update edit))
```
Comment (snayrouz) [inline app/services/permission_service.rb:43]: good update.

File: app/controllers/store/admin/store_controller.rb (hunk 1 above)
Comment (sidewinder2020) [reply app/controllers/store/admin/store_controller.rb:8]: lol, always a )

**Review by snayrouz — APPROVED** [review]
approved!

**Review by djtrujillo — APPROVED** [review]

**Pull request merged**